singularity-eos/closure/mixed_cell_models.hpp

```diff
@@ -1148,6 +1183,15 @@ class PTESolverPT
     return ResidualNorm();
   }
 
+  PORTABLE_INLINE_FUNCTION
+  void Finalize() {
+    for (std::size_t m = 0; m < nmat; m++) {
+      press[m] = Pequil * uscale;
+      temp[m] = Tequil * Tnorm;
+      u[m] *= uscale;
+    }
+  }
```

Comment:
This was a bug in the PTESolverPT. If the solver was passed a PTE solution, it would not overwrite the pressures with `Pequil` and they might be uninitialized memory. This ensures that can't happen.